content/en/docs/setup/minikube.md

```diff
@@ -9,146 +9,206 @@ content_template: templates/concept
+	```
+	If the output shows the `STATUS` as `Running`, the Pod is now up and running:
+	```
+	NAME                              READY     STATUS    RESTARTS   AGE
+	hello-minikube-3383150820-vctvh   1/1       Running   0          13s
+	```
+5. Get the URL of the exposed Service to view the Service details:
+	```shell
+	minikube service hello-minikube --url
+	```
+6. To view the details of your local cluster, copy and paste the URL you got as the output, on your browser.
+    
+    The output is similar to this:
+    ```
+    Hostname: hello-minikube-7c77b68cff-8wdzq
+
+    Pod Information:
+        -no pod information available-
+
+    Server values:
+        server_version=nginx: 1.13.3 - lua: 10008
+
+    Request Information:
+        client_address=172.17.0.1
+        method=GET
+        real path=/
+        query=
+        request_version=1.1
+        request_scheme=http
+        request_uri=http://192.168.99.100:8080/
+
+    Request Headers:
+        accept=*/*
+        host=192.168.99.100:30674
+        user-agent=curl/7.47.0
+
+    Request Body:
+        -no body in request-
+    ```
+	If you no longer want the Service and cluster to run, you can delete them.
+7. Delete the `hello-minikube` Service:
+    ```shell
+    kubectl delete services hello-minikube
+    ```
+    The output is similar to this:
+    ```
+    service "hello-minikube" deleted
+    ```
+8. Delete the `hello-minikube` Deployment:
+    ```shell
+    kubectl delete deployment hello-minikube
+    ```
+    The output is similar to this:
+    ```
+    deployment.extensions "hello-minikube" deleted
+    ```
+9. Stop the local Minikube cluster:
+    ```shell
+    minikube stop
+    ```
+    The output is similar to this:
+    ```
+    Stopping "minikube"...
+    "minikube" stopped.
+    ```
+	For more information, see [Stopping a Cluster](docs/setup/minikube/#stopping-a-cluster).
+10. Delete the local Minikube cluster:
+    ```shell
+    minikube delete
+    ```
+    The output is similar to this:
+    ```
+    Deleting "minikube" ...
+    The "minikube" cluster has been deleted.
+    ```
+	For more information, see [Deleting a cluster](/docs/setup/minikube/#deleting-a-cluster).
+  
+## Managing your Cluster
 
-We can see that the pod is still being created from the ContainerCreating status
-kubectl get pod
+### Starting a Cluster
 
-```
-NAME                              READY     STATUS    RESTARTS   AGE
-hello-minikube-3383150820-vctvh   1/1       Running   0          13s
-```
+The `minikube start` command can be used to start your cluster.
+This command creates and configures a Virtual Machine that runs a single-node Kubernetes cluster.
+This command also configures your [kubectl](/docs/user-guide/kubectl-overview/) installation to communicate with this cluster.
 
-We can see that the pod is now Running and we will now be able to curl it:
+If you are behind a web proxy, you will need to pass this information to the `minikube start` command:
```

Comment:
How about putting this in a note section? Is that appropriate?

Reply:
Eliminate future tense:
> If you are behind a web proxy, you must pass this information...